core/spec/dummy/app/models/author.rb

```diff
@@ -1,7 +1,3 @@
 class Author < ActiveRecord::Base
   has_many :blogs
-
-  def to_s
-    name
-  end
```

Comment:
SimpleForm tries to call a couple of methods among which is `name`, which happens to be an attribute of Author.